lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -369,20 +353,16 @@ class ImageElements extends FRGatherer {
       ],
     });
 
+    elements.sort((a, b) => estimateFileSize(b) - estimateFileSize(a))
```

Comment:
nit: let's just accept we have to sort by image area. just amounts to renaming this function.